css/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-color: #191765;
+  font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;
+}
```

Comment:
- [ ] Your page font looks different from your chosen template. I suggest you use `Roboto` font-family like this `font-family: 'Roboto', sans-serif;` and see if it fixes the font so that you have the same font as that of your template.
- [ ] Please also consider using the correct margin on all the content of your page as they are different from the template.